src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -315,6 +315,10 @@ public override ImmutableArray<CSharpAttributeData> GetAttributes()
 
         internal override bool IsMetadataOptional => _baseParameterForAttributes?.IsMetadataOptional == true;
 
+        internal override bool IsMetadataIn => base.IsMetadataIn || _baseParameterForAttributes?.IsMetadataIn == true;
+
+        internal override bool IsMetadataOut => base.IsMetadataOut || _baseParameterForAttributes?.IsMetadataOut == true;
```

Comment:
>base.IsMetadataOut || _baseParameterForAttributes?.IsMetadataOut == true [](http://example.com/codeflow?start=48&length=72)

Are we testing both branches here?

Reply:
Adding a test which includes a custom attribute to ensure we test this branch (or its equivalent in the most recent revision).